src/main/java/edu/cornell/kfs/sys/dataaccess/DocumentstoreClient.java

```diff
@@ -0,0 +1,5 @@
+package edu.cornell.kfs.sys.dataaccess;
+
+public interface DocumentstoreClient {
```

Comment:
It might be better to rename this "DocumentStoreClient" (capitalize "Store"), for consistency with some other base KFS classes with "DocumentStore" in their names.